### PR TITLE
Added circle-help and user icon

### PR DESCRIPTION
**Pull request by TheAlphamerc**
## I have read the [CONTRIBUTING.md](https://github.com/pqoqubbw/icons/blob/main/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Feature: 2 Icon added

#  Demo
### Help Icon
https://github.com/user-attachments/assets/8c902662-9a29-45a7-b7b1-a88201cc46a7

### User Icon

https://github.com/user-attachments/assets/496b876d-3d57-4d20-93d1-fafb44b59829

### CODE REVIEW

File: icons/user.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    pathOffset: [1, 0],
+  },
+};
+
+const circleVariant: Variants = {
+  normal: {
+    pathLength: 1,
+    opacity: 1,
+    pathOffset: 0,
+    scale: [1],
```
Comment (pqoqubbw) [inline icons/user.tsx:20]:
```suggestion
    scale: 1,
```

File: icons/user.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    pathLength: [0, 1],
+    opacity: [0, 1],
+    pathOffset: [1, 0],
+  },
+};
+
+const circleVariant: Variants = {
+  normal: {
+    pathLength: 1,
+    opacity: 1,
```
Comment (pqoqubbw) [inline icons/user.tsx:18]: i guess you don't need `opacity` here cause you don't change it

Comment (TheAlphamerc) [reply icons/user.tsx]: Opacity Removed

File: icons/user.tsx (hunk 1 above)
Comment (TheAlphamerc) [reply icons/user.tsx]: Done

**Pull request merged**